rep-2000.rst

```diff
@@ -1,20 +1,69 @@
 We define platforms to include both operating system releases (e.g. Ubuntu Xenial (16.04 LTS)) as well as major language releases (e.g. Python 3.5).
 The target platforms represent the set on which all core stacks are expected to work.
 Exceptions can be made for stacks that are intentionally platform-specific.
 
+Release Schedule
+================
+
+Note: the following applies to ROS 2 releases *after* Foxy Fitzroy.
+Before, releases were made more frequently but with shorter support due to the fact that many foundational parts of ROS 2 were still heavily iterated on.
+
+TL;DR ROS 2 release schedule post Foxy:
+* May 2021: G Turtle: non-LTS release, supported for 1.5 years
+* May 2022: H Turtle: LTS release, supported for 5 years
+* May 2023: I Turtle: non-LTS release, supported for 1.5 years
+* May 2024: J Turtle: LTS release, supported for 5 years
+* and so on, alternating annually between LTS and non-LTS releases
+
+Frequency
+---------
+
+New ROS 2 releases are published in a time based fashion **every 12 months**.
+The rationale is that a shorter cycle (like 6 months) results in a significant overhead and many active releases at the same time (assuming they have the same support length).
+On the other hand a longer cycle (like 2 years) is too long for users to wait for new features to be available in a ROS 2 release.
+
+Targeted Platforms
+------------------
+
+Since regular Ubuntu releases are only supported for 9 months ROS 2 is not targting those.
+A single ROS 2 distribution only **targets one Ubuntu LTS**.
+The rationale is that supporting two LTS versions - which means 2-year-different versions of upstream dependencies - is a tremendous overhead and sometimes even impossible.
+
+Since macOS (or at least brew) as well as Windows are rolling platforms we aim to support the latest version available at the time of a ROS 2 distribution release.
+For Debian we also aim to target the latest stable version though if that version is two years behind the Ubuntu version that might not be possible.
+
+Support
+-------
+
+LTS releases
+~~~~~~~~~~~~
+
+Since Ubuntu LTS releases come with **5 years** of standard support we aim to match the same duration.
+In even years new ROS 2 releases happen shortly (a month) after the Ubuntu LTS has been released (some time in May).
+The ROS 2 release will be supported until the end of the standard support of the Ubuntu LTS release which is 4 years and 11 months from the ROS 2 release date.
+
+Non-LTS Releases
+~~~~~~~~~~~~~~~~
+
+If every ROS 2 release would have that support we would end up with 5 active releases at the same time which would pose a significant maintenance effort.
+To lower that effort while still providing releases more frequently than every two years in odd years a non-LTS ROS release is being published.
+It will target the same Ubuntu LTS as the previous ROS LTS release but is only supported for **1.5 years**.
```

Comment:
Ah I think this answers my question. Still might be worth clarifying above, though.

Reply:
See my reply on the other comment: https://github.com/ros-infrastructure/rep/pull/238#discussion_r385864893